pom.xml

```diff
@@ -114,6 +84,149 @@
             <version>4.8.0</version>
             <scope>runtime</scope>
         </dependency>
```

Comment:
Minor: Can you move these 3 dependencies with the other runtime dependencies to keep everything in the same place?

Reply:
Moved